test/e2e/node/config.go

```diff
@@ -23,7 +23,6 @@ type Config struct {
 	PrivValServer    string                      `toml:"privval_server"`
 	PrivValKey       string                      `toml:"privval_key"`
 	PrivValState     string                      `toml:"privval_state"`
-	Misbehaviors     map[string]string           `toml:"misbehaviors"`
```

Comment:
So this doesn't currently support misbehavior? (Doesn't look like this was used anywhere)

Reply:
Yeah I think this was vestige from v0.34 back when the maverick node was around. It's unused.